### PR TITLE
Addition of individual parameter manipulation for the WhatsApp Unoapi Provider

### PR DESCRIPTION
**Pull request by wkarts**
# Pull Request Template

## Description

This PR includes the addition of individual parameter manipulation for the WhatsApp Unoapi Provider. The parameters added include:

- Reject call
- Message when rejecting call
- Send agent name
- Ignore group messages
- Ignore history on connect
- Send messages to webhook
- Ignore status broadcast
- Ignore broadcast messages
- Ignore own messages
- Ignore your messages
- Show connection status
- Notify messages with failure
- Compose message
- Show reaction as a reply
- Display profile picture
- Generate API Key

Files altered:
- `app/javascript/dashboard/i18n/locale/en/inboxMgmt.json`
- `app/javascript/dashboard/i18n/locale/pt_BR/inboxMgmt.json`
- `app/javascript/dashboard/routes/dashboard/settings/inbox/channels/Unoapi.vue`
- `app/javascript/dashboard/routes/dashboard/settings/inbox/settingsPage/UnoapiConfiguration.vue`
- `app/services/whatsapp/unoapi_webhook_setup_service.rb`

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

- Unit tests were added to cover the new parameters.
- Manual testing was conducted to verify the correct behavior of each parameter in different scenarios.
- End-to-end testing was performed to ensure integration with the Unoapi Provider works as expected.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/dashboard/routes/dashboard/settings/inbox/settingsPage/UnoapiConfiguration.vue [hunk 1]
```diff
@@ -174,13 +382,21 @@ export default {
...
-        this.inbox.provider_config.ignore_group_messages;
-      this.ignoreHistoryMessages =
-        this.inbox.provider_config.ignore_history_messages;
-      this.webhookSendNewMessages =
-        this.inbox.provider_config.webhook_send_new_messages;
+      this.ignoreGroupMessages = this.inbox.provider_config.ignore_group_messages;
+      this.ignoreHistoryMessages = this.inbox.provider_config.ignore_history_messages;
+      this.webhookSendNewMessages = this.inbox.provider_config.webhook_send_new_messages;
       this.sendAgentName = this.inbox.provider_config.send_agent_name;
+      this.ignoreBroadcastStatuses = this.inbox.provider_config.ignore_Broadcast_Statuses;
```
Comment (clairton) [inline app/javascript/dashboard/routes/dashboard/settings/inbox/settingsPage/UnoapiConfiguration.vue]: alterar por favor de `ignore_Broadcast_Statuses` para `ignore_broadcast_statuses` e assim com as outras propriedades =)

**Review by clairton — COMMENTED** [review]
@wkarts muito obrigado pela contribuição, vai ser muito útil, só peço para alterar a grafia as configurações onde tem o _, a próxima é minuscula, por exemplo `ignore_Broadcast_Statuses` para `ignore_broadcast_statuses` pois é usado desse forma no json, no padrão snake_case, enquanto no javascript é camelCase

File: app/javascript/dashboard/routes/dashboard/settings/inbox/settingsPage/UnoapiConfiguration.vue (hunk 1 above)
Comment (wkarts) [reply app/javascript/dashboard/routes/dashboard/settings/inbox/settingsPage/UnoapiConfiguration.vue]: @clairton muito obrigado por aceitar a contribuição. 
apenas modificar: app/javascript/dashboard/routes/dashboard/settings/inbox/settingsPage/UnoapiConfiguration.vue
image

ou também devo aplicar à: app/services/whatsapp/unoapi_webhook_setup_service.rb
image
image

**Review by clairton — APPROVED** [review]
thank's @wkarts

**Pull request merged**